SPECS/gnupg2/gnupg2.spec

```diff
@@ -89,6 +89,10 @@ ln -s $(pwd)/bin/gpg $(pwd)/bin/gpg2
 %defattr(-,root,root)
 
 %changelog
+* Mon Jan 23 2023 CBL-Mariner Servicing Account <cblmargh@microsoft.com> - 2.4.0-1
+- Auto-upgrade to 2.4.0 - to fix CVE-2022-3515
```

Comment:
Can we please modify the PR title to reflect that we fixed a CVE by bumping to 2.4.0.
Something like:
`Upgrade gnupg2 to v2.4.0 to address CVE-2022-3515`